src/main/java/net/snowflake/client/jdbc/SFAsyncResultSet.java

```diff
@@ -364,1082 +359,4 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
     raiseSQLExceptionIfResultSetIsClosed();
     throw new SQLFeatureNotSupportedException();
   }
-
```

Comment:
All of these same functions can be accessed in SnowflakeResultSetV1 instead. No need for the duplicate here.